docs/post-upgrade/update-deprecated-sn-connector.asciidoc

```diff
@@ -0,0 +1,58 @@
+[[cases-update-deprecated-sn-connector]]
+== Update a deprecated ServiceNow connector
+
+{sn} ITSM and SecOps connectors created in 7.15 or earlier are marked as deprecated after you upgrade to 7.16. Deprecated connectors have a yellow icon after their name and display a warning message when you select them.
+
+[role="screenshot"]
+image::images/cases-deprecated-sn-connector.png[]
+
+IMPORTANT: Deprecated connectors will continue to function with the rules they were added to and can be assigned to a new rule. Deprecated connectors will also continue to function with the cases that they were assigned to, but cannot be assigned to a new case. We strongly recommend updating the deprecated connector or <<creating-new-connector, creating a new one>> to ensure you have access to connector enhancements, such as updating incidents.
```

Comment:
"a new rule" -> "new rules"

Reply:
"a new case" -> "new cases"

Reply:
"recommend updating the deprecated connector or creating a new one" -> "recommend updating deprecated connectors, or creating new ones"